internal/apiserver/server.go

```diff
@@ -185,7 +175,16 @@ func (s *Server) UpdateUser(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
-	resp, err := s.grpcServer.UpdateUser(context.Background(), &api.UserObject{Id: uint32(id), Name: name, Email: email, Age: uint32(age)})
+	if (name == "") || (email == "") || (age == 0) {
```

Comment:
should you please this logic in some external validation function. 

or create a type `User` with function aside `models.UserFromHTTPReq(r *http.Request)` + method `IsValid()`